src/main/drivers/bus_i2c_stm32f10x.c

```diff
@@ -387,8 +387,8 @@ void i2cInit(I2CDevice device)
     IO_t scl = IOGetByTag(i2c->scl);
     IO_t sda = IOGetByTag(i2c->sda);
 
-    IOInit(scl, OWNER_I2C_SCL, RESOURCE_INDEX(device));
-    IOInit(sda, OWNER_I2C_SDA, RESOURCE_INDEX(device));
+    IOInit(scl, OWNER_I2C_SCL, (uint8_t)RESOURCE_INDEX(device));
+    IOInit(sda, OWNER_I2C_SDA, (uint8_t)RESOURCE_INDEX(device));
```

Comment:
Yeah, but all usage may not use `int` arg and expect `uint8_t` in return. Or? Did not dare to hack into resource.h just because of this, leads to far and may affect others. Wildfire spreading.